CollapseLauncher/XAMLs/MainApp/Pages/HomePage.xaml.cs

```diff
@@ -1019,12 +1022,12 @@ private async void CheckRunningGameInstance(CancellationToken Token)
                             IGameSettingsUniversal gameSettings = CurrentGameProperty!._GameSettings!.AsIGameSettingsUniversal();
                             PresetConfig gamePreset = CurrentGameProperty._GamePreset;
 
-                            StartPlaytimeCounter(currentGameProcess, gamePreset, fromActivityOffset);
+                            CurrentGameProperty!._GamePlaytime!.StartSession(currentGameProcess);
```

Comment:
I think we can move the StartSession call into CheckRunningGameInstance so the playtime can be tracked if the game is started before Collapse does

Reply:
isn't that call already inside CheckRunningGameInstance?

Reply:
Oh ye.... My bad didn't see it xd
All good then. We'll wait for neon's check and all good